SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/WorkerRegion.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Components/StaticMeshComponent.h"
+#include "GameFramework/Actor.h"
+#include "Math/Box2D.h"
+#include "Math/Color.h"
+
+#include "WorkerRegion.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogWorkerRegion, Log, All)
+
+UCLASS()
```

Comment:
could add the keyword here that prevents spawning type type through the editor, and prevents it being subclassed in the editor, unless you expect those use cases.  NotBlueprintable and NotPlaceable.